### ISSUE TITLE
how to switch coins every x amount of minutes  

### ISSUE THREAD

**Issue by amosmc81** [bonesoul/CoiniumServ#944]
hello all im trying to figure out how i can switch coins every 15 mins or so or if its possible i want to set my kids up a pool on different coins to get them started into mining and teach them a little is this possible

**Comment by jeong760** [comment]
I don't think so CoiniumServ support a profit switch option like nomp or unmop. CoiniumServ also does not support a merged mining option.

**Comment by amosmc81** [comment]
yea i dont want to do a profit switch just switch coins like every 15 mins in rotation regardless of profit
